Mock is upstream returns 501 (#1426)
stoplightio/prism: CHANGELOG.md

# Unreleased

## Changed

> - When running in proxy mode and an upstream server responds with `501`, Prism will now "remock" the request and provide a meaningful response. This is a fancy way to say "if the upstream server does not respond, it will mock" [#1426](https://github.com/stoplightio/prism/pull/1426)

Reviewer: ```suggestion
- When running in proxy mode and an upstream server responds with `501`, Prism will now "remock" the request and provide a meaningful response. This is a fancy way to say "if the upstream server has not yet implemented an operation, it will mock" [#1426](https://github.com/stoplightio/prism/pull/1426)
```
